stable/kubernetes-dashboard/templates/clusterrole-readonly.yaml

```diff
@@ -0,0 +1,160 @@
+{{- if and .Values.rbac.create (not .Values.rbac.clusterAdminRole) }}
```

Comment:
maybe an and for .Values.rbac.clusterReadonlyRole? For those that aren't using it, shouldn't need it loaded.

Reply:
Makes sense, thank you.

Reply:
Changed to `{{- if and .Values.rbac.create .Values.rbac.clusterReadOnlyRole (not .Values.rbac.clusterAdminRole) }}`; squashed into the first commit.